docs/developer-guide/etendo-classic/concepts/Element_and_Synchronize_Terminology.md

```diff
@@ -113,6 +111,10 @@ If you observe the fields in the `Application Dictionary` > `Setup` >`Element`
 you will notice there are standard fields (name, help...) and similar ones
 but for purchases.
 
+In Etendo Classic there are some tables that are used for sales and purchases
+transactions, for example _C_Invoice_ table contains both sales and purchases
+invoices. This means that _Purchase Invoice || Header_ and _Sales Invoice ||
+Header_ tabs use the same table. But it is very usual to label fields for the
```

Comment:
"label fields for the" what?
 This piece of text is duplicated, in the lines below you have the same text